scripts/quick-start-koku.sh

```
@@ -365,36 +423,27 @@ kokuVEnv() {
   $PIPENV_BIN run pre-commit install
 }
 
-# Start the full koku backend stack via podman compose and wait up to 3 minutes
-# for the API to become healthy. Applies compose overrides to work around two
-# Podman quirks: port ranges and legacy 'links:' directives.
+# Start the full koku backend stack via podman compose and wait for the API.
+# Applies a compose override for a Podman quirk with worker port ranges.
 kokuServices() {
   cd $KOKU_DIR
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)